site/docs/components/tabs/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Tabs
+data:
+  description:
+    Tabs switch between different but related content. Users can move between different views without the need to leave the current page.
+    A tabstrip comprises a minimum of two tabs, with one tab actively engaged at any given moment. The active tab is differentiated from an inactive tab by a colored indicator either on the top or bottom edge of the tab, depending on the variant in use. These tabstrips can expand the full width of the page or be used within content areas, or within components such as dialogs, cards, and drawers.
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/src/tabs-next/TabNext.tsx"
+  package:
+    name: "@salt-ds/lab"
+  alsoKnownAs: ["Tab", "Tabstrip", "Tab bar", "Tab set"]
+  relatedComponents: [{ name: "Navigation item", relationship: "similarTo" }]
+  stickerSheet: ""
```

Comment:
does this need a `status: Lab component` ?

Reply:
The mosaic plugin uses package name to determine this now.
https://github.com/jpmorganchase/salt-ds/blob/main/site/src/mosaic-plugins/LabsComponentPlugin.mjs